src/exceptions/bad-request.exception.ts

```diff
@@ -0,0 +1,7 @@
+﻿export class BadRequestException extends Error {
+  public readonly status = 400;
+
+  constructor(message: string = `Неправильно выполнен запрос`) {
```

Comment:
Лучше на английском оставить +- оригинальные названия
```suggestion
  constructor(message: string = `Bad Request`) {
```